src/test/ui/consts/const-eval/issue-51300.rs

```diff
@@ -1,4 +1,4 @@
-// build-pass (FIXME(62277): could be check-pass?)
+// build-pass
```

Comment:
Why did you make this build-pass?

Reply:
(And there are some more.)

Reply:
I re-read all of the `build-pass` ones, and I took the time to do more research about the issues they prevent regression to. All in all I changed the all to `check-pass`